lib/base/bed.mli

```diff
@@ -3,7 +3,7 @@ type parser_error = [ `Bed_parser_error of int * string ]
 
 type item = string * int * int * string list
 
-val item_of_line : Line.t -> (item, string) result
+val item_of_line : Line.t -> (item, string) Caml.result
```

Comment:
I'm not sure why you need that. From `base`'s `Result` module I read:
```ocaml
type ('ok, 'err) t = ('ok, 'err) Pervasives.result =
  | Ok of 'ok
  | Error of 'err
[@@deriving_inline sexp, compare, hash]
```

Reply:
@pveber ```lib/base/bed.mli|6 col 45 warning| Error (warning 3): deprecated: Base.result [2016-09] this element comes from the stdlib distributed with OCaml. Referi
    ng to the stdlib directly is discouraged by Base. You should either use the equivalent functionality offered by Base, or if you really want to refer to t
    he stdlib, use Caml.result instead [ocaml/merlin]
~
~